<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LicenseEditor</class>
 <widget class="QWidget" name="LicenseEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>350</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <layout class="QFormLayout" name="infoLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="isValidTitle">
       <property name="text">
        <string>授权有效</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="isValid">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="deviceSerialTitle">
       <property name="text">
        <string>授权序列号</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="deviceId">
       <property name="text">
        <string/>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>授权类型</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="deviceType">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>授权描述</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="deviceDesc">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="customerTitle">
       <property name="text">
        <string>授权给</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="licenseCustomer">
       <property name="text">
        <string/>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="typeTitle">
       <property name="text">
        <string>设备型号</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="licenseModel">
       <property name="text">
        <string/>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="serialTitle">
       <property name="text">
        <string>设备序列号</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="licenseSerial">
       <property name="text">
        <string/>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="dateTitle">
       <property name="text">
        <string>出厂日期</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="licenseDate">
       <property name="text">
        <string/>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="deviceTimeTitle">
       <property name="text">
        <string>当前时间</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="deviceTime">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QTreeWidget" name="levels">
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>授权</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>到期</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>到期时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>激活</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>状态</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="update">
       <property name="text">
        <string>写入设备</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="recharge">
       <property name="text">
        <string>更新授权</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generateRecharge">
       <property name="text">
        <string>生成授权码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="editConfig">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/icons/Fatcow.qrc">
         <normaloff>:/fatcow/16x16/cog_edit.png</normaloff>:/fatcow/16x16/cog_edit.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addLevel">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../Resources/Resources.qrc">
         <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removeLevel">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../Resources/Resources.qrc">
         <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/icons/Fatcow.qrc"/>
  <include location="../../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
